src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrPatchesTestsBase.cs

```diff
@@ -16,6 +16,7 @@ public class VmrPatchesTestsBase : VmrTestsBase
 {
     protected string patchFileName = null!;
     protected LocalPath installerPatchesDir = null!;
+    protected LocalPath installerFilePath = null!;
```

Comment:
I noticed this already when writing my tests - protected members should be properties and capitalized.

So ideally:
```csharp
protected LocalPath InstallerFilePath { get; private set; } = null!;
```